Source/Scene/BingMapsImageryProvider.js

```diff
@@ -103,10 +104,23 @@ define([
         }
         //>>includeEnd('debug');
 
+        if (defined(options.proxy)) {
+            deprecationWarning('BingMapsImageryProvider.proxy', 'The options.proxy parameter has been deprecated. Specify options.url as a Resource instance and set the proxy property there.');
+        }
+
         this._key = BingMapsApi.getKey(options.key);
         this._keyErrorCredit = BingMapsApi.getErrorCredit(options.key);
 
-        this._url = options.url;
+        var urlResource = Resource.createIfNeeded(options.url, {
+            proxy: options.proxy
+        });
+
+        urlResource.addQueryParameters({
+            key: this._key
```

Comment:
Adding the key here results in it being sent with every tile request, which is not correct.  The key is only supposed to be sent with the metadata request, as it was in the previous code.  I'll open a PR.